{
  "produtos": [
    {
      "Id": 0,
      "Application": "FaturamentoPromax",
      "Enviroment": "",
      "Date": "2022-05-18T17:08:06.0832942+00:00",
      "Thread": "nsdfuasfujasdf",
      "Level": "Error",
      "Logger": "",
      "Message": "An exception has been raised that is likely due to a transient failure.",
      "Exception": "System.InvalidOperationException"
    }
, {
  "Id": 1,
  "Application": "FaturamentoPromax",
  "Enviroment": "",
  "Date": "2022-05-18T17:08:06.0832942+00:00",
  "Thread": "   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at FaturamentoPromax.Dados.UnitOfWork.CommitAsync() in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Dados/UnitOfWork.cs:line 60\n   at FaturamentoPromax.Negocio.Services.PedidoEmFaturamentoService.ReenviarPedidosParaFaturamento(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Negocio/Services/PedidoEmFaturamentoService.cs:line 59\n   at FaturamentoPromax.WebApi.Controllers.PedidoController.Refaturar(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.WebApi/Controllers/PedidoController.cs:line 37\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)",
  "Level": "Info",
  "Logger": "",
  "Message": "An exception has been raised that is likely due to a transient failure.",
  "Exception": "System.InvalidOperationException"
}, {
  "Id": 2,
  "Application": "FaturamentoPromax",
  "Enviroment": "",
  "Date": "2022-05-18T17:08:06.0832942+00:00",
  "Thread": "   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at FaturamentoPromax.Dados.UnitOfWork.CommitAsync() in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Dados/UnitOfWork.cs:line 60\n   at FaturamentoPromax.Negocio.Services.PedidoEmFaturamentoService.ReenviarPedidosParaFaturamento(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Negocio/Services/PedidoEmFaturamentoService.cs:line 59\n   at FaturamentoPromax.WebApi.Controllers.PedidoController.Refaturar(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.WebApi/Controllers/PedidoController.cs:line 37\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)",
  "Level": "Warning",
  "Logger": "",
  "Message": "usahfuihasfiasjf",
  "Exception": "System.InvalidOperationException"
}, {
  "Id": 3,
  "Application": "FaturamentoPromax",
  "Enviroment": "",
  "Date": "2022-05-18T17:08:06.0832942+00:00",
  "Thread": "   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at FaturamentoPromax.Dados.UnitOfWork.CommitAsync() in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Dados/UnitOfWork.cs:line 60\n   at FaturamentoPromax.Negocio.Services.PedidoEmFaturamentoService.ReenviarPedidosParaFaturamento(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Negocio/Services/PedidoEmFaturamentoService.cs:line 59\n   at FaturamentoPromax.WebApi.Controllers.PedidoController.Refaturar(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.WebApi/Controllers/PedidoController.cs:line 37\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)",
  "Level": "Error",
  "Logger": "",
  "Message": "An exception has been raised that is likely due to a transient failure.",
  "Exception": "System.InvalidOperationException"
}, {
  "Id": 4,
  "Application": "FaturamentoPromax",
  "Enviroment": "",
  "Date": "2022-05-18T17:08:06.0832942+00:00",
  "Thread": "   at Npgsql.EntityFrameworkCore.PostgreSQL.Storage.Internal.NpgsqlExecutionStrategy.ExecuteAsync[TState,TResult](TState state, Func`4 operation, Func`4 verifySucceeded, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at Microsoft.EntityFrameworkCore.DbContext.SaveChangesAsync(Boolean acceptAllChangesOnSuccess, CancellationToken cancellationToken)\n   at FaturamentoPromax.Dados.UnitOfWork.CommitAsync() in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Dados/UnitOfWork.cs:line 60\n   at FaturamentoPromax.Negocio.Services.PedidoEmFaturamentoService.ReenviarPedidosParaFaturamento(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.Negocio/Services/PedidoEmFaturamentoService.cs:line 59\n   at FaturamentoPromax.WebApi.Controllers.PedidoController.Refaturar(Guid guidEnvio) in /home/vsts/work/1/s/FaturamentoPromax/backend/src/FaturamentoPromax.WebApi/Controllers/PedidoController.cs:line 37\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)\n   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextExceptionFilterAsync>g__Awaited|26_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)",
  "Level": "Warning",
  "Logger": "",
  "Message": "An exception has been raised that is likely due to a transient failure.",
  "Exception": "System.InvalidOperationException"
}


   
  ]
}